web/profiles/custom/nysenate/config/local/email_tfa.settings.yml

```diff
@@ -5,7 +5,7 @@ subject: 'One Time Password'
 body: "Dear [user:name],\r\n    \r\nYour One Time Passcode for completing your NY Senate TFA Verification is: [user:email_tfa]\r\n    \r\nPlease use this Passcode to complete your transaction. Do not share this Passcode with anyone.\r\n    \r\nThank you,\r\n[site:name] team"
 routes: "email_tfa.verifiy\r\nuser.logout"
 timeouts: 300
-status: false
+status: true
```

Comment:
Is this turning on TFA for local environments?

Reply:
@routinet It seems that way to me. I wonder if it was just a mistake testing locally. I think it would be safe to change to `false`. Would you like me to make that change?

Reply:
Our goal is to ensure TFA is always enabled on `live` for all privileged users (e.g., LC, MCP, admins).  `test` should _probably_ have TFA enabled, though the reroute_email address should be set to our dev gmail account (nys.senate.dev@gmail.com).  For all other environments, TFA should be disabled.